libraries/bsg_manycore.h

```diff
@@ -293,6 +293,19 @@ extern "C" {
         /* DMA API */
         /***********/
 
+        /**
```

Comment:
Not really sure what changed here.

Reply:
Oh you removed static - purposefully? I would put it back.

Reply:
This is a nice api function to check if npa is a dram npa. I wanted to check that before invoking DMA. Do we need this to be static?

Reply:
It's defined in a header outside of a class declaration - it needs to be static otherwise you can run into linker errors.

Reply:
Oh the linker error, yes. Done!